packages/in_app_purchase/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.0+4
+
+* Fix a UserInfo types on NSError.
```

Comment:
Nits:
```suggestion
* iOS: Support unsupported UserInfo value types on NSError.
```

Reply:
@cyanglaz 
Hi. Thank you for your quick review.
it fixed ab59ee9